src/example-types/text.js

```diff
@@ -7,8 +7,19 @@ let joinmap = {
   none: '$nor',
 }
 
+// Convert to an array, strip empty strings, and determine if there are any values
+let hasValue = _.flow(
+  // NOTE: Don't change the below, otherwise things will explode!
+  x => F.cascade(['value', 'values'], x),
```

Comment:
I think this can be just:
```
let hasValue = _.flow(
  F.cascade(['value', 'values']),
  _.castArray,
  _.without(['', undefined]),
  _.negate(_.isEmpty)
)
```

Seems to pass all unit tests etc https://runkit.com/akrion/hasvalue

Reply:
Actually, that's the whole bug/solution. That's why I added the comment above. Easier to explain over a Slack call or at stand-up.

Reply:
Yeah - we should include a link to the relevant issue in the comment: https://github.com/smartprocure/futil-js/issues/218

Reply:
I added the comment myself so this doesn't have to stay held up